js/src/pages/dashboard/index.js

```diff
@@ -83,6 +84,7 @@ const Dashboard = () => {
 
 	return (
 		<>
+			<ExperienceRatingBanner />
```

Comment:
Is there a reason this sits outside the `gla-dashboard` div below rather than as a child of that `div`?

Reply:
No apparent reason. Switched it to inside the div and everything seems to be fine.

Reply:
Updated in 6b879a0e234